app/lpc55xpresso/app.toml

```diff
@@ -113,11 +113,11 @@ task-slots = ["gpio_driver", "syscon_driver"]
 [tasks.rng_driver]
 name = "drv-lpc55-rng"
 priority = 3
-max-sizes = {flash = 16384, ram = 4096}
 uses = ["rng", "pmc"]
 start = true
-stacksize = 2200
+stacksize = 4200
```

Comment:
So I think this'd make it the highest stack usage in the system, above control-plane-agent. Do we have a sense for what caused this? (Asking because I've been winding up playing stack size whack-a-mole lately.)

Reply:
Having read more of the code, I can answer this question at least partially: the `Server` type used in the RNG server is large, and it's being created on the stack and passed around.

It's large _primarily_ due to the `Sha3` embedded within it, which I recently noticed in an unrelated case was about 2 kiB.

The easiest way to fix this would be to allocate a static Sha3 instance using `ClaimOnceCell` (from `lib/static-cell`) and then pass it into the server constructor to take ownership of (via a `&'static mut`).

Reply:
Had to use `mutable-statics` because `Sha3` doesn't have a const constructor (per discussion in chat): [b43295c](https://github.com/oxidecomputer/hubris/pull/1820/commits/b43295cc9a8bf2f7668a17916d6c86d7d40becfc)